sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.net462.cs

```diff
@@ -120,3 +120,46 @@ public ProvisionableTemplateAttribute(string resourceName) { }
         public string ResourceName { get { throw null; } }
     }
 }
+namespace System.ClientModel
+{
+    public partial class AccessToken
+    {
+        public AccessToken(string tokenValue, string tokenType, System.DateTimeOffset expiresOn, System.DateTimeOffset? refreshOn = default(System.DateTimeOffset?)) { }
+        public System.DateTimeOffset ExpiresOn { get { throw null; } protected set { } }
+        public System.DateTimeOffset? RefreshOn { get { throw null; } protected set { } }
+        public string TokenType { get { throw null; } protected set { } }
+        public string TokenValue { get { throw null; } protected set { } }
+    }
+    public abstract partial class RefreshableToken : System.ClientModel.AccessToken
+    {
+        public RefreshableToken(string tokenValue, string tokenType, System.DateTimeOffset expiresOn, System.DateTimeOffset? refreshOn = default(System.DateTimeOffset?)) : base (default(string), default(string), default(System.DateTimeOffset), default(System.DateTimeOffset?)) { }
+        public abstract System.Threading.Tasks.Task RefreshAsync(System.Threading.CancellationToken cancellationToken);
+    }
+}
+namespace System.ClientModel.Auth
+{
+    public partial class OAuth2BearerTokenAuthenticationPolicy : System.ClientModel.Primitives.PipelinePolicy
+    {
+        public OAuth2BearerTokenAuthenticationPolicy(System.ClientModel.Auth.TokenProvider tokenProvider, System.Collections.Generic.IEnumerable<System.Collections.Generic.IReadOnlyDictionary<string, object>> contexts) { }
+        public override void Process(System.ClientModel.Primitives.PipelineMessage message, System.Collections.Generic.IReadOnlyList<System.ClientModel.Primitives.PipelinePolicy> pipeline, int currentIndex) { }
+        public override System.Threading.Tasks.ValueTask ProcessAsync(System.ClientModel.Primitives.PipelineMessage message, System.Collections.Generic.IReadOnlyList<System.ClientModel.Primitives.PipelinePolicy> pipeline, int currentIndex) { throw null; }
+    }
+    public partial class TokenFlowProperties
+    {
+        public const string AuthorizationUrlPropertyName = "authorizationUrl";
```

Comment:
are we absolutely sure there will not ever be a reason to change the values? If yes, const is fine. If there is a slightest chance, we should make them read-only static fields.

Reply:
These values are only for consistency in the property bag. They have no effect on what is sent over the wire, and so there should be no reason for them to change.